src/helpers/coveragepy.ts

```diff
@@ -0,0 +1,11 @@
+import { isProgramInstalled, runExternalProgram } from "./util"
+import { UploadLogger } from './logger'
+
+export async function generateCoveragePyFile(): Promise<string> {
+    if (!isProgramInstalled('coverage')) {
+        UploadLogger.verbose(`coveragepy is not installed, skipping coverage xml conversion`)
+        return '';
```

Comment:
```suggestion
        throw new Error('coveragepy is not installed, skipping coverage xml conversion')
```

https://github.com/codecov/uploader/blob/469569cd1d39e906225010318051294eb2923f2f/src/helpers/xcode.ts#L12

Reply:
A try-catch may be of use back in index.ts where the function is called.